components/brave_new_tab_ui/components/default/braveNews/index.tsx

```diff
@@ -129,8 +129,9 @@ export default function BraveNewsSection(props: Props) {
       }
       {shouldDisplayContent && <React.Suspense fallback={(<CardLoading />)}>
         {defaultState.featureFlagBraveNewsPromptEnabled
-          ? <FeedV2 />
```

Comment:
In my head, these two flags were the same, so I helpfully refactored this :facepalm: